source/documentation/xsd-reference.html.md.erb

```diff
@@ -20,41 +20,43 @@ Use these to validate your XML. Please note, there are some known omissions. The
 
 You can ignore this field.
 
-### [Download zip file](ctc-traders-api-xsds.zip) with all the CTC Traders API XSDs.
+### [Download zip file](https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-service-guide/documentation/ctc-traders-api-xsds.zip) with all the CTC Traders API XSDs.
```

Comment:
Can we make the url relative?  If it contains the hostname, it probably will not work in QA.

Reply:
So `./documentation/ctc-traders-api-xsds.zip`?